### PR TITLE
Remove some unforking TODOs

### PR DESCRIPTION
**Pull request by mattverse**
Closes: #XXX

## What is the purpose of the change
Remove some unforking TODOs

## Testing and Verifying

This change is a trivial rework / code cleanup without any test coverage.

## Documentation and Release Note

  - [ ] Does this pull request introduce a new feature or user-facing behavior changes?
  - [ ] Changelog entry added to `Unreleased` section of `CHANGELOG.md`?

Where is the change documented?
  - [ ] Specification (`x/{module}/README.md`)
  - [ ] Osmosis documentation site
  - [ ] Code comments?
  - [ ] N/A

### CODE REVIEW

File: x/gamm/keeper/gas_test.go [hunk 1]
```diff
@@ -74,8 +74,6 @@ func (s *KeeperTestSuite) TestJoinPoolGas() {
 	))
 
 	firstJoinGas := s.measureJoinPoolGas(defaultAddr, poolId, minShareOutAmount, defaultCoins)
-	// UNFORKINGNOTE: This used to be capped at LessOrEqual to 100000, but unforking increased this value.
```
Comment (mattverse) [inline x/gamm/keeper/gas_test.go:77]: Gas change is due to logic change in bank module (bank send)

File: x/ibc-hooks/sdkmodule.go [hunk 2]
```diff
@@ -23,7 +23,6 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
-// UNFORKING v2 TODO: I added these to every module, verify they all apply to every module
```
Comment (mattverse) [inline x/ibc-hooks/sdkmodule.go:26]: Applies to every module

File: x/gamm/keeper/gas_test.go (hunk 1 above)
Comment (czarcas7ic) [reply x/gamm/keeper/gas_test.go:77]: Do you know what extra logic is done in bank sends that causes this increase?

**Review by czarcas7ic — APPROVED** [review]

File: x/gamm/keeper/gas_test.go (hunk 1 above)
Comment (mattverse) [reply x/gamm/keeper/gas_test.go:77]: I don't I can specifically debug and track down which causes this will get back to you

Comment (PaddyMc) [reply x/gamm/keeper/gas_test.go:77]: @mattverse any updates here?

Comment (mattverse) [reply x/gamm/keeper/gas_test.go:77]: Apologies for the delay here, https://github.com/osmosis-labs/cosmos-sdk/blob/osmo/v0.50.x/x/bank/keeper/send.go#L246-L250 it's this specific logic that got added to send

**Review by PaddyMc — APPROVED** [review]

**Pull request merged**